src/eda.py

```diff
@@ -15,27 +15,11 @@
 import os
```

Comment:
Are we using this package anywhere? If no, can we remove it?

Reply:
yes, only one line: while checking if the path is valid or not.